api/spec/spec_helper.rb

```diff
@@ -15,6 +15,8 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 require "fakeweb"
+# Library for stubbing and setting expectations on HTTP requests in Ruby.
+require 'webmock/rspec'
```

Comment:
If you're using webmock just for this purpose: `FakeWeb.allow_net_connect = false`
We don't need both.
Btw, I've switched all test domains to non routable TLD